Configuration/TCA/tx_calendarize_domain_model_index.php

```diff
@@ -8,6 +8,15 @@
 
 $base = ModelUtility::getTcaInformation(Index::class);
 
+// Removed unused fields, because the records are not accessible by the user
+unset(
```

Comment:
I am not sure of the unset here - even if fields are not accessible by the user they can still have TCA definitions. In some cases this is advisable to set default values. If the editlock is not accessible, shouldn't it get removed entirely?

Reply:
The editlock field is added by EXT:autoloader. In EXT:autoloader it is part of the "enableFields" and we cannot disable a single field out of this set: https://github.com/lochmueller/autoloader/blob/master/Classes/DataSet/EnableFields.php

Reply:
I missed that point. The column definition of editlock can also be removed.
```php
unset(
    $base['columns']['editlock'],
);
```
---
The reindex in only required, if multiple languages are used.